source/html/renderer/src/utils.ts

```diff
@@ -27,22 +27,39 @@ export function isNullOrEmpty(value: string): boolean {
     return value === undefined || value === null || value === "";
 }
 
-export function stringEqualIgnoreCase(str1:string, str2:string): boolean{
-    if(isNullOrEmpty(str1) || isNullOrEmpty(str2)) return str1 === str2;
-    return str1.toLowerCase() === str2.toLowerCase();
-}
-
 export function appendChild(node: Node, child: Node) {
     if (child != null && child != undefined) {
         node.appendChild(child);
     }
 }
 
+export function getEnumValueOrDefault(targetEnum: { [s: number]: string }, name: string, defaultValue: number): number {
+    if (isNullOrEmpty(name)) {
+        return defaultValue;
+    }
+
+    for (var key in targetEnum) {
+        let isValueProperty = parseInt(key, 10) >= 0
```

Comment:
I've been trying to use "var" everywhere instead of "let" (although there are still some left.) Both will work just fine, and actually we should probably use let everywhere rather than var.

Reply:
Agreed. Will gradually change to let/const.